src/main/cpp/events/baseaudioevent.h

```diff
@@ -134,15 +134,8 @@ class BaseAudioEvent
         // ( 1, 32, 4 ) positions audioEvent at 4 / 32 = 1/8th note in the second measure
         virtual void positionEvent( int startMeasure, int subdivisions, int offset );
 
-        /* looping (e.g. for samples) */
-
-        virtual bool isLoopeable();
-        virtual void setLoopeable( bool value );
-
```

Comment:
Looping is really a concern of SampleEvents only. Move this logic out of _BaseAudioEvent_ and into the _SampleEvent_ class.